src/aes_key.c

```diff
@@ -190,8 +188,8 @@ zpc_aes_key_set_type(struct zpc_aes_key *aes_key, int type)
 		aes_key->napqns = 0;
 		aes_key->apqns_set = 0;
 
-		rc = __aes_key_alloc_apqns_from_mkvp(&(aes_key->apqns),
-		    &(aes_key->napqns), aes_key->mkvp, type);
+		rc = alloc_apqns_from_mkvp(pkeyfd, &(aes_key->apqns), &(aes_key->napqns),
```

Comment:
I noticed that originally all internal helper functions were named __xxxx with 2 leading underscores. You seem to have given up this naming scheme..... Is this intentionally?

Reply:
Yes, I didn't always continue with the previous naming scheme. Especially for this routine the decision was: previously there was the __aes_key_alloc_apqns_from_mkvp. For ECC I first introduced a corresponding __ec_key_alloc_apqns_from_mkvp. But this routine was completely identical to the aes routine. So I consolidated the two into alloc_apqns_from_mkvp in pkey.c where no other routine has the "__" prefix.